### PR TITLE
Updating Animating.gi

### PR DESCRIPTION
**Pull request by TomGoertzen**
I create a new Method SetVertexCoordinatesNC which takes 3 argument;
Replaced IsCyclomatic by IsPosInt,
Replaced IsSimplicialSurface and IsNotEdgeRamified by IsSimplicialSurface;
And updated various descriptions

### CODE REVIEW

File: gap/PolygonalComplexes/animating.gi [hunk 1]
```diff
@@ -1,78 +1,117 @@
...
-				printRecord.vertexCoordinates3D := coordinates;
-				return printRecord;
+	printRecord.vertexCoordinates3D := coordinates;
+        return printRecord;
+    end
+);
+InstallOtherMethod( SetVertexCoordinates3DNC,
+    "for a simplicial surface, for a list of coordinates and a record",
+    [IsDenseList, IsRecord],
+    function(coordinates, printRecord)
```
Comment (markusbaumeister) [inline gap/PolygonalComplexes/animating.gi:13]: You need to add the surface to the argument lists.

**Review by markusbaumeister — CHANGES_REQUESTED** [review]
Apart from one comment, this looks fine.

**Review by markusbaumeister — APPROVED** [review]
This looks fine now :).

**Review by MeikeWeiss — CHANGES_REQUESTED** [review]
Looks good, but there are some merge conflicts. Can you resolve them?

**Review by MeikeWeiss — APPROVED** [review]

**Pull request merged**